README.md

```diff
@@ -67,6 +67,7 @@ This collection provides a list of educational resources for Software Engineers.
 - [CS50 Web](https://cs50.harvard.edu/web/2020/)- Diving more deeply into the design and implementation of web apps with Python, JavaScript, and SQL.
 - [Codementor](https://www.codementor.io/web-dev-experts)- Improve your coding skills or get help with an ongoing web development project.
 - [HTML elements PDF](https://html.com/wp-content/uploads/html-cheat-sheet.pdf) - Become a HTML geek by finding definitions, examples and references clearly with this PDF.
+-[Online Tutorials] (https://www.youtube.com/watch?v=weZFfrjF-k4&list=RDCMUCbwXnUipZsLfUckBPsC7Jog&index=2) - Find projects on HTML, CSS, and Javascript on this youtube page
```

Comment:
Do changes suggested by @HareerSakhi

Reply:
Thanks @MRslimcoder 😀

Reply:
@HareerSakhi can you please share your email want to discuss something related to this project.

Reply:
Could we please discuss it here?

Reply:
Yeah! so keep your more focus on this repo, review more and more PR's  also send more PR's, 
If you wanr to contribute more then checkour this organization: https://github.com/devcreatives

Reply:
@AvidCoder101, @MRslimcoder  done